libraries/SRV_Channel/SRV_Channel.cpp

```diff
@@ -122,13 +131,18 @@ void SRV_Channel::calc_pwm(int16_t output_scaled)
     } else {
         pwm = pwm_from_range(output_scaled);
     }
+    // If not in unlimited ouput we contrain the ouput
+    if (unlimited_pwm_mask & (1U << ch_num)) {
```

Comment:
> the mask is set when at 0

@khancyr that's contrary to the common definition: when a bit is set, its value is 1. So setting something in a mask means setting it to 1, not to 0.
I'm with @WickedShell, the variable needs to be named *limited_pwm_mask*.